Monika After Story/game/script-topics.rpy

```diff
@@ -13165,3 +13166,30 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_songwriting", 
+            category=["music"],
+            prompt="Songwriting", 
+            random=True, 
+            pool=True
+        )
+    )
+
+label monika_songwriting:
+    m 3eub "Hey [player], have you ever written a song?"
+    m 3eua "I've tried myself, and it's pretty fun. It takes a bit of time to do, but the end product is definitely worth it!"
+    m 4eub "You know, I used to write songs all the time in my sophomore years. I used to be pretty good!"
```

Comment:
Still need to double check those facts I'd imagine.

Reply:
Well, you can't really say for sure whether that was true or not, since they never really talked much about Monika before the events of the game in DDLC. But then again, I guess it would probably be false, or at least just scripted, from what Monika has said in the "Tell me about yourself" topic.
![screenshot0002](https://user-images.githubusercontent.com/52469676/66261272-2ac35380-e798-11e9-95b5-004946fe4c08.png)

Reply:
If I recall, Monika only wanted to learn piano after meeting the player, so she scrambled to start learning piano to put something together in just a few days, so I'm leaning more towards no, this isn't the case.